### PR TITLE
split logger to new debug and new dev loggers with different levels

### PR DESCRIPTION
**Pull request by Kschappacher**

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -102,6 +102,33 @@ func NewLoggerConfigForGCP() zap.Config {
 
 // NewDevelopmentLoggerConfig returns a new default development logger config.
 func NewDevelopmentLoggerConfig() zap.Config {
+	// from https://github.com/uber-go/zap/blob/2314926ec34c23ee21f3dd4399438469668f8097/config.go#L135
+	// but disable stacktraces, use same keys as prod, and color levels.
+	return zap.Config{
```
Comment (edaniels) [inline logger.go]: this should use new debug logger but change level

File: logger.go [hunk 2]
```diff
@@ -154,6 +181,15 @@ func NewDevelopmentLogger(name string) Logger {
 	return logger.Sugar().Named(name)
 }
 
+// NewDevelopmentLogger returns a new logger using the default development configuration.
```
Comment (edaniels) [inline logger.go:166]: fix comment

**Review by edaniels — CHANGES_REQUESTED** [review]
!

File: logger.go (hunk 1 above)
Comment (Kschappacher) [reply logger.go]: used debug config and changed level and returned the new config

File: logger.go (hunk 2 above)
Comment (Kschappacher) [reply logger.go:166]: fixed

**Review by edaniels — APPROVED** [review]
AMAZING

**Pull request merged**